lib/src/http/managers/sticker_manager.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:nyxx/src/builders/builder.dart';
+import 'package:nyxx/src/http/managers/manager.dart';
+import 'package:nyxx/src/http/request.dart';
+import 'package:nyxx/src/http/route.dart';
+import 'package:nyxx/src/models/snowflake.dart';
+import 'package:nyxx/src/models/snowflake_entity/snowflake_entity.dart';
+import 'package:nyxx/src/models/sticker/sticker.dart';
+import 'package:nyxx/src/utils/parsing_helpers.dart';
+
+class StickerManger extends Manager<Sticker> {
+  final Snowflake? guildId;
+
+  StickerManger(super.config, super.client, {required this.guildId}) : super(identifier: guildId != null ? "$guildId.stickers" : "stickers");
+
+  @override
+  WritableSnowflakeEntity<Sticker> operator [](Snowflake id) => PartialSticker(id: id, manager: this);
```

Comment:
```suggestion
  PartialSticker operator [](Snowflake id) => PartialSticker(id: id, manager: this);
```

This should be typed as a `PartialSticker`